index.html

```diff
@@ -132,34 +150,7 @@ <h5>Scheduled for February 3rd and 4th, 2024.</h5>
               -moz-osx-font-smoothing: grayscale;
             "
             target="_blank"
-            >Register
-            <div class="icon-typeform"></div>
-          </a>
-          <a
-            id="typeFormButtonStyleLarge"
-            class="typeform-share button button-link"
-            href="https://46ttvu3ju7i.typeform.com/to/QH3I3hv0"
-            data-mode="popup"
-            style="
-              text-decoration: none;
-              cursor: pointer;
-              font-family: Montserrat, Arial, sans-serif;
-              line-height: 65px;
-              text-align: center;
-              margin: 0;
-              height: 65px;
-              padding: 0px 43px;
-              border-radius: 32px;
-              max-width: 100%;
-              white-space: nowrap;
-              overflow: hidden;
-              text-overflow: ellipsis;
-              font-weight: bold;
-              -webkit-font-smoothing: antialiased;
-              -moz-osx-font-smoothing: grayscale;
-            "
-            target="_blank"
-            >Register to Attend
+            >Apply to Attend
```

Comment:
i think from all these changes that you may have not pulled before making changes. i would rebase and see if that helps.